mqtt_client.c

```diff
@@ -990,6 +992,30 @@ static esp_err_t deliver_publish(esp_mqtt_client_handle_t client)
     return ESP_OK;
 }
 
+static esp_err_t deliver_suback(esp_mqtt_client_handle_t client)
+{
+    uint8_t *msg_buf = client->mqtt_state.in_buffer;
+    size_t msg_read_len = client->mqtt_state.in_buffer_read_len;
+    size_t msg_data_len = msg_read_len;
+    char *msg_data = NULL;
+    ESP_LOGD(TAG, "Get data len= %zu, read len= %zu", msg_data_len, msg_read_len);
```

Comment:
Would suggest removing this log. We already have one in the client's code and might not want to pollute debug logs too much.